src/storage/db/backup.ts

```diff
@@ -1,26 +1,26 @@
-import { getEncodedToken, type Proof, type Token } from '@cashu/cashu-ts'
+// import { getEncodedToken, type Proof, type Token } from '@cashu/cashu-ts'
```

Comment:
File fully commented out. I feel the pain of throwing away code :(

Reply:
It will be used again but not as "backup".

It is a great function written by @BilligsterUser